packages/backend/src/app.ts

```diff
@@ -393,7 +394,21 @@ export class App {
     )
 
     let tenantApiSignatureResult: TenantApiSignatureResult
-    if (this.config.env !== 'test') {
+    if (this.config.env === 'test') {
+      const tenantService = await this.container.use('tenantService')
+      let tenant = await tenantService.get(this.config.operatorTenantId)
+      if (!tenant) {
+        tenant = await Tenant.query().insertAndFetch({
```

Comment:
I really don't like this. Do you guys have an alternative suggesetion? 
The "re-insert" is necessary due to the tenant table being cleared, then the `TenantedApolloContext` does not have a tenant.

Reply:
Can you elaborate on why we need to do this? 

Also, dont we need to insert some initial operator for non test environments? Like when initiallty deploying these changes? Im fuzzy on that but thought we were either going to seed it with some default values which need to be updated, or inserting somewhere like here.

If that is the case, I'm just wondering if we can piggy off that here. Insert some default operator which can be updated by the integrator or in the tests.

Reply:
> The "re-insert" is necessary due to the tenant table being cleared

Oh, are we talking about how `truncateTables` clears out everything for the tests? You could add the tenant table to the ignore list inside that function.

Reply:
I would also suggest with this approach to have a `truncateTables` call in the `afterAll` block that doesn't ignore it, so it gets cleaned up eventually

Reply:
Ok, I agree. So to confirm: 
1. The `20241205153035_seed_operator_tenant.js` will create the tenant as part of migration script;
2. The `truncateTables` will be updated to include `tenant` as one of the ignored tables

Reply:
Updated.